lib/components/artist/artworks/index.js

```diff
@@ -23,6 +24,13 @@ class Artworks extends React.Component {
     this.state = { completedForSaleWorks: false }
   }
 
+  onFetchMore(cb = () => {}) {
+    console.log('onfetchmore, this', this)
+    this.props.relay.setVariables({
+      pageSize: this.props.relay.variables.pageSize + PageSize
```

Comment:
The `pageSize` variable should probably be renamed to reflect that it’s the _total_ amount of entries fetched.